src/controllers/heroController.js

```diff
@@ -21,6 +22,21 @@ class HeroController {
       res.status(400).json({ error: 'Please provide a username and a userId' });
     }
   }
+
+  getHeroById(req, res) {
+    const heroId = Number(req.params.heroId);
+    if (heroId) {

```

Comment:
if(0){} else{console.log(1)} -> there will be a log
use if (!Number.isNaN(heroId)) instead

Reply:
Awesome, fixed, cheers.